docs/_documentations/debugging.md

```diff
@@ -18,6 +18,10 @@ Complete the following steps to debug in Codewind applications:
 4. While the project is still in Debugging mode, if the debug process is detached, right-click your project and select **Attach Debugger** from the menu. This action reattaches the debugger to the debug process, and you can continue with debugging your application.
 5. To switch back to run mode, right-click your project and select **Restart in Run Mode** from the menu.
 
+## Debugging remotely
```

Comment:
Did you see the comments I added to https://github.com/eclipse/codewind/issues/2651 including some Eclipse specific information?

Reply:
Oh--I'll take a look at that! Thank you!